src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -144,7 +144,7 @@ public static function register($mode = false)
                 if (!empty($notices)) {
                     echo "\n";
                 }
-                if ('weak' !== $mode && ($deprecations['unsilenced'] || $deprecations['remaining'] || $deprecations['other'])) {
+                if ('weak' !== $mode && 'weak-verbose' !== $mode && ($deprecations['unsilenced'] || $deprecations['remaining'] || $deprecations['other'])) {
```

Comment:
we should add those strings as constants esp. since $mode is part of the public api